### PR TITLE
Draft js complete cleanup

### PR DESCRIPTION
**Pull request by oded-soffrin**

### CODE REVIEW

**Comment by oded-soffrin** [comment]
This branch attempts to remove all draft-js references OTHER than ricos-content where it's still needed for converting
@danarish4 many occurrences in plugin, probably will be removed when done with plugin cleanup
@lxgreen some references are editor related

@talevy17 @tamrat-bay FYI

File: examples/main/src/text-color-style-fn.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import type { EditorProps } from 'draft-js';
+import type { EditorProps } from 'draft-js'; //TODO
```
Comment (lxgreen) [inline examples/main/src/text-color-style-fn.ts:1]: i guess this file can be deleted?

File: packages/ricos-content/web/package.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
     },
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|svg)$": "<rootDir>/__mocks__/fileMock.js",
-      "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js",
+      "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js"
       "^draft-js$": "<rootDir>/../../../node_modules/@wix/draft-js",
       "^draft-js/": "<rootDir>/../../../node_modules/@wix/draft-js/"
```
Comment (lxgreen) [inline packages/ricos-content/web/package.json:76]:
```suggestion
```

File: packages/ricos-content/web/src/comparision/compare.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /** Based on https://gist.github.com/Yimiprod/7ee176597fef230d1451 */
 /* eslint-disable no-console, @typescript-eslint/no-explicit-any, fp/no-delete */
-import type { RawDraftEntityRange } from '@wix/draft-js';
+import type { RawDraftEntityRange } from '@wix/draft-js'; //TODO
```
Comment (lxgreen) [inline packages/ricos-content/web/src/comparision/compare.ts:3]: still needed in compare as we use the `compare` in all the draft converter tests

File: examples/main/src/text-color-style-fn.ts (hunk 1 above)
Comment (oded-soffrin) [reply examples/main/src/text-color-style-fn.ts:1]: you mean `customStyleFn` is no longer supported ?

Comment (oded-soffrin) [reply examples/main/src/text-color-style-fn.ts]: I see it's supported but the type is wrong, i've fixed it

**Pull request merged**